types.ts

```diff
@@ -213,18 +213,17 @@ export interface RelationshipPerson {
   id: number;
   firstName: string;
   lastName: string;
+  gender?: string;
 }
 
 export interface Relationship {
-  parents: Array<RelationshipPerson>;
-  siblings: Array<RelationshipPerson>;
-  children: Array<RelationshipPerson>;
-  other: Array<RelationshipPerson>;
+  type: string;
+  persons: RelationshipPerson[];
```

Comment:
`people`?

Reply:
that is what comes from the backend response :)